### ISSUE TITLE
Sonification for skateboard

### ISSUE THREAD

**Issue by samreid** [phetsims/forces-and-motion-basics#151]
We have discussed adding a "click/clack" sound for the skateboard, so that the frequency of clicks/clacks is proportional to the speed of the skateboard.  @ariel-phet do you want to do this for the next publication of Forces and Motion Basics?

Main questions:
1. Does the sound need to visually synchronize with the look of the "cracks" in the sidewalk?
2. How much should it sound like a real skateboard?
3. Does the sound change as a function of friction (on ice, etc.) or as a function of what is on the skateboard?

**Comment by ariel-phet** [comment]
@samreid - no, we should not include this on the next redepoly.

I think we only want to include such sonifiied representations in published sims after they have been researched and vetted.

**Comment by amanda-phet** [comment]
Deferring this for when the design of the sim has more dedicated time.

**Comment by samreid** [comment]
I don't think it makes sense to keep this issue open. If/when we do full sonification for this sim, we can do this if desired.
